lib/account/account-service.js

```diff
@@ -1,99 +1,140 @@
 "use strict";
 
 /**
- * What we initially send back to the client so they can start to initiate
- * the complete procedure and save information.
+ * What we initially generate in the registration part of storage
+ * till the user has confirmed and completed the account
  *
  * @typedef {Object} AccountService~initiateResult
- * @property {string} accountId
+ * @property {string} regId
  * @property {string} email
  * @property {string} mfaKey
- * @property {string} salt
+ * @property {string} passwordSalt
  */
 
 /**
- * Stored information for the account.  This is changed from the initiated
+ * Stored information for the account. This is changed from the initiated
  * file as we add password and there could be other items added after
  * we initiate the account.
```

Comment:
Same comment here.  This is a finalized account record.  Try to not focus on what happens to it.